### ISSUE TITLE
utxoCostPerWord in the ProtocolParameters needs to be renamed to utxoCostPerByte

### ISSUE THREAD

**Issue by Jimbo4350** [input-output-hk/cardano-node#4052]

**Comment by Jimbo4350** [comment]
In the Babbage era

**Comment by gitmachtl** [comment]
@JaredCorduan can the current tags/1.35.0 handle the change to utxoCostPerByte? will utxoCostPerWord go away with the change or will it stay for a while in parallel?

**Comment by JaredCorduan** [comment]
@gitmachtl the semantic change in the protocol is attached to the Babbage era, and will go into effect at the moment of the hardfork.

The `cardano-api`, however, is still translating the new Babbage parameter `coinsPerUTxOByte` into a "per word" value:

https://github.com/input-output-hk/cardano-node/blob/42809ad3d420f0695eb147d4c66b90573d27cafd/cardano-api/src/Cardano/Api/ProtocolParameters.hs#L1743

In other words, 1.35 is ready to move to the new min-utxo calculation when the hardfork happens, but the CLI will still convert the protocol parameter to be measuring words (not bytes), even though the new calculation will be used. This issue is to not make this conversion.

**Comment by gitmachtl** [comment]
ah ok thx, this explains why the miniprotocol is sending the CostPerByte value, thx!

**Comment by lehins** [comment]
If I understand correctly this renaming also needs to be applied to `ProtocolParametersUpdate`:

https://github.com/input-output-hk/cardano-node/blob/0cd6878769a10097e7c780e15efb9919fb94f669/cardano-api/src/Cardano/Api/ProtocolParameters.hs#L1304-L1305

**Comment by gitmachtl** [comment]
And the removal of the *8 😄
